openmetadata-ui/src/main/resources/ui/src/components/Settings/Team/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -267,14 +262,15 @@ const TeamDetailsV1 = ({
               must_not: [
                 {
                   term: {
-                    'name.keyword': 'Organization',
+                    'name.keyword': currentTeam.name,
```

Comment:
I believe the `Organization` team was put here intentionally, so that it won't be returned in the search result. Any reason for changing that to `currentTeam.name`?